tests/regression_tests/benchmark/test_model_performance.py

```diff
@@ -0,0 +1,111 @@
+import os
+from dataclasses import dataclass
+from enum import Enum
+from typing import List, Union
+
+import pytest
+from dataclasses_json import dataclass_json
+
+from ludwig.benchmarking.benchmark import benchmark
+from ludwig.constants import MODEL_ECD, MODEL_GBM
+from ludwig.utils.data_utils import load_json, load_yaml
+
+BENCHMARKING_CONFIG = """
+experiment_name: {experiment_name}
+hyperopt: false
+export:
+  export_artifacts: true
+  export_base_path: {export_base_path}
+profiler:
+  enable: false
+  use_torch_profiler: false
+  logging_interval: 0.1
+experiments:
+  - dataset_name: {dataset}
+    config_path: {config_path}
+"""
+
+
+class MetricRegressionDirection(Enum):
```

Comment:
Can implement this using existing logic in Ludwig metrics: https://github.com/ludwig-ai/ludwig/blob/f000e73ea1d3a28d32575c066437a26db10607c7/ludwig/modules/metric_modules.py#L120

Reply:
Using `abs(expected - actual) < tolerance` instead of minimize/maximize.